### DISCUSSION TITLE
Where are log files written to on Windows?

### DISCUSSION THREAD

**Discussion by SuperDuperDan22** [akosbalasko/yarle#603] [Q&A] [answered]
Yarle crashes on a certain note.  If I can look at logs I can figure out which one, but not sure where to find logs.  The logs showing in the log window is not accessible due to the app crashing.

**Comment by SuperDuperDan22** [comment, marked as answer]
Nm, found it: C:\Users\<username>\AppData\Roaming\yarle-evernote-to-md\conversion.log

**Reply by akosbalasko** [reply]
@SuperDuperDan22  did you find the root cause of the crash?

**Reply by SuperDuperDan22** [reply]
Yes, just a bad note.  I deleted the note in Evernote and re-exported to notebook to .enex without the note.  It then worked fine.

Not sure what it was about the note, but it also caused Obsidian to crash when I tried to import directly using the Import tool and in fact, also caused Evernote itself to stop responding for about a minute when I clicked on the note.

**Reply by akosbalasko** [reply]
okok, thank you. Strange.

**Reply by spudart** [reply]
Does anyone know where the log file is stored for MacOS?

**Reply by spudart** [reply]
I found the location for the log on MacOS: 
`/Users/username/Library/Application Support/yarle-evernote-to-md/conversion.log`
